plugins/cache_promote/lru_policy.h

```diff
@@ -104,9 +104,16 @@ class LRUPolicy : public PromotionPolicy
     return "LRU";
   }
 
+  const std::string
+  id() const override
+  {
+    return _label + ";LRU=b:" + std::to_string(_buckets) + ",h:" + std::to_string(_hits);
```

Comment:
Do these values change? Might set this up in the constructor and then return the member.

Reply:
No, those do not change, they are part of the LRU objects, the value are set and defined via the @pparam configurations. The label is used to make sure that we can modify the settings  in remap.config, such that the --label is the same, but e.g. the bucket size is changed.

I can save it in the constructor if you feel strongly, there's no performance impact here, but I see your point.